sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/VersionsHelper.java

```diff
@@ -68,14 +69,14 @@ private List<ArtifactVersion> getVersions(Artifact artifact){
 
     public List<ArtifactVersion> getAllVersions(Artifact artifact, int maxSize) {
         List<ArtifactVersion> versions = getVersions(artifact);
+        versions.removeIf(version -> "3.1.0-SNAPSHOT".equals(version.toString()));
+        versions.removeIf(version -> "3.0.0-SNAPSHOT".equals(version.toString()));
```

Comment:
Ooh... I don't exactly like doing this because we'd need to constantly maintain this. Is there a way we can write some logic that would